test-browser/helpers/contracts.js

```diff
@@ -21,7 +21,7 @@ function testContracts (browser, contractCode, compiledContractNames, callback)
       .clearValue('#input textarea')
       .click('.newFile')
       .setValue('#input textarea', contractCode, function () {})
-      .waitForElementPresent('.contract .create', 5000, true, function () {
+      .waitForElementPresent('.contract .create', 15000, true, function () {
```

Comment:
i'm not sure.
I got random fails when running the tests... sometimes stuff passes, sometimes it doesnt.
Putting it to 15 sec. stopped the problem for me